harper-core/src/linting/oxford_comma.rs

```diff
@@ -150,4 +150,13 @@ mod tests {
             "They enjoy playing soccer, basketball, or tennis.",
         );
     }
+
+    #[test]
+    fn nor_vegetables() {
+        assert_suggestion_result(
+            "I like carrots, kale nor broccoli.",
+            OxfordComma::default(),
+            "I like carrots, kale, nor broccoli.",
```

Comment:
I feel like these should have neither at the beginning, but it's gramatically correct as is.